libHttpClient.props

```diff
@@ -1,66 +1,101 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--Project
+    ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
+    InitialTargets="HCBuildDebug"
+-->
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup Condition="'$(ApplicationType)'=='Windows Store'">
-    <HCLibPlatformType>UWP</HCLibPlatformType>
-    <libHttpClientDefine>HC_UWP_API</libHttpClientDefine>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(ApplicationType)'=='' AND '$(Platform)'!='Durango'">
-    <HCLibPlatformType>Win32</HCLibPlatformType>  
-    <libHttpClientDefine Condition="'$(libHttpClientDefine)' == ''">HC_WIN32_API</libHttpClientDefine>
-    <useWsppWebsockets Condition="'$(useWsppWebsockets)' == ''">true</useWsppWebsockets>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(ApplicationType)'=='' AND '$(Platform)'=='Durango'">
-    <HCLibPlatformType>XDK</HCLibPlatformType>
-    <libHttpClientDefine>HC_XDK_API</libHttpClientDefine>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(ApplicationType)'=='Android'">
-    <HCLibPlatformType>Android</HCLibPlatformType>
-    <libHttpClientDefine>HC_ANDROID_API</libHttpClientDefine>
-    <useWsppWebsockets>true</useWsppWebsockets>
-  </PropertyGroup>
```

Comment:
Why is this removed?  We are now relying on clients to define HCLibPlatformType which seems worse that inferring it here.

Reply:
For both of these comments I want to be sure the settings are consistent across the board (openssl, libhttpclient, xal, xsapi all have to agree). Right now manually setting the values in each project guarantees that.

In a subsequent change, we can create a shared props file that does the guessing, and all projects use it.